local/config/finna/datasources.ini.sample

```diff
@@ -125,3 +125,7 @@ video_sources[] = "application/dash+xml|https://link.to.video/{videoname}.mpd"
 allowArchiveRequest = true
 archiveRequestAllowedRecordLevels = 'file:item'
 archiveRequestRequireFilingUnit = true
+
+; Hide record fields from record view
```

Comment:
Tässä voisi mainita jotain tyyliin "Each value is a field label defined in RecordDataFormatterFactory".